test/snapshots/windows/squirrelWindowsTest.js.snap

```diff
@@ -33,19 +35,23 @@ exports[`Squirrel.Windows 2`] = `
   "lib/net45/LICENSES.chromium.html",
   "lib/net45/resources.pak",
   "lib/net45/snapshot_blob.bin",
-  "lib/net45/Test App ßW.exe",
-  "lib/net45/Test App ßW_ExecutionStub.exe",
-  "lib/net45/Update.exe",
+  "lib/net45/squirrel.exe",
+  "lib/net45/test with spaces.exe",
+  "lib/net45/test with spaces_ExecutionStub.exe",
   "lib/net45/v8_context_snapshot.bin",
   "lib/net45/vk_swiftshader.dll",
   "lib/net45/vk_swiftshader_icd.json",
   "lib/net45/vulkan-1.dll",
   "lib/net45/locales/en-US.pak",
+  "lib/net45/resources/",
   "lib/net45/resources/app.asar",
-  "lib/net45/swiftshader/libEGL.dll",
-  "lib/net45/swiftshader/libGLESv2.dll",
```

Comment:
QQ: Why are these no longer present?

Reply:
Nice catch! Thanks for adding it to the test suite. Does it need to be its own test-runner in the matrix though? can we add it to another runner?

Looks like squirrel tests take 4min. We can merge it with the `winCodeSignTest,differentialUpdateTest` test runner since that one only takes 2ish minutes.

Reply:
Agreed. Fixed.